pkg/datapath/loader/base.go

```diff
@@ -196,14 +197,27 @@ func (l *Loader) reinitializeIPSec(ctx context.Context) error {
 
 	// No interfaces is valid in tunnel disabled case
 	if len(interfaces) != 0 {
```

Comment:
Nit: I'd make this an early return and unindent the whole thing.

Reply:
Ah so close. I thought about doing this and then discarded :disappointed: